packages/next/bin/next.ts

```diff
@@ -75,6 +76,10 @@ if (args['--help']) {
   forwardedArgs.push('--help')
 }
 
+if (args['--page']) {
+  args['--page'].forEach((page) => forwardedArgs.push('--page', page))
```

Comment:
Is this needed? The argument should be part of forwardedArgs already I think 🤔 or is it because of the array support?